src/app/ApplicationTemplate.Access/ApplicationTemplate.Access.csproj

```diff
@@ -21,7 +21,7 @@
 		<PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="7.0.1" />
 		<PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="3.1.0" />
 		<PackageReference Include="Reactive.Annex" Version="2.0.0" />
-		<PackageReference Include="Refit" Version="7.0.0" />
+		<PackageReference Include="Refit" Version="8.0.0" />
 		<PackageReference Include="ReviewService.Abstractions" Version="1.0.0" />
 		<PackageReference Include="System.Text.Json" Version="8.0.5" />
 		<PackageReference Include="MallardMessageHandlers" Version="1.2.0" />
```

Comment:
You'll also want to update `MallardMessageHandlers` to 2.0.0 (which should be available soon now that your PR merged).